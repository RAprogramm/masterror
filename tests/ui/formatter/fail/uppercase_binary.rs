use masterror::Error;

#[derive(Debug, Error)]
#[error("{value:B}")]
struct UppercaseBinary {
    value: u8,
}

fn main() {}
