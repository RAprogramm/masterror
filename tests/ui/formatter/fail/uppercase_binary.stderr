error: placeholder spanning bytes 0..9 uses an unsupported formatter
 --> tests/ui/formatter/fail/uppercase_binary.rs:4:9
  |
4 | #[error("{value:B}")]
  |         ^^^^^^^^^^^

error: missing #[error(...)] attribute
 --> tests/ui/formatter/fail/uppercase_binary.rs:5:8
  |
5 | struct UppercaseBinary {
  |        ^^^^^^^^^^^^^^^
